tests/lib/tools/report-mongodb

```diff
@@ -69,6 +70,15 @@ def insert(db, col, value):
         print("report-mongodb: invalid value, it does not contain a valid json format")
         sys.exit(1)
 
+    try:
+        date = json_value["date"]
+        time = json_value["time"]
+        created_at = datetime.fromisoformat("{} {}".format(date, time)).isoformat()
+    except:
```

Comment:
This is a very broad exception handler, what is the typcial exception we are expecting here? Maybe we can make it a bit more targeted?

Reply:
@mvo5 thanks for the review, I think it is ValueError expected there, I'll update that.